### PR TITLE
Enhance stereo depth develop merge 

### PR DESCRIPTION
**Pull request by iamsg-luxonis**
Enhance stereo depth develop merge

### CODE REVIEW

File: include/depthai/device.hpp [hunk 1]
```diff
@@ -36,6 +36,12 @@ class Device{
     );
     std::vector<std::string> get_available_streams();
 
+    std::vector<std::vector<float>> get_left_intrinsic();
+    std::vector<std::vector<float>> get_left_homography();
+    std::vector<std::vector<float>> get_right_intrinsic();
+    std::vector<std::vector<float>> get_right_homography();
+    std::vector<std::vector<float>> get_rotation();
+    std::vector<float> get_Translation();
```
Comment (themarpe) [inline include/depthai/device.hpp:44]: Rename to `get_translation`

File: include/depthai/host_data_packet.hpp [hunk 2]
```diff
@@ -37,7 +37,10 @@ struct HostDataPacket
         // Check if metadata is valid
         if(metadata.isValid()){
             // copy only frame data
-            frameSize = metadata.frameSize;
+            if (stream_name_ == "disparity_color") // FIXME see if applicable to all streams
```
Comment (themarpe) [inline include/depthai/host_data_packet.hpp:40]: metadata.frameSize should be set correctly. Maybe rather fix over at device side instead?

**Review by themarpe — CHANGES_REQUESTED** [review]

File: include/depthai/host_data_packet.hpp (hunk 2 above)
Comment (iamsg-luxonis) [reply include/depthai/host_data_packet.hpp:40]: "disparity_color" is a virtual stream, created on the host ("disparity" colorized on host)

Comment (themarpe) [reply include/depthai/host_data_packet.hpp:40]: Can you add the metadata to the frame while creating it and then send it onwards to "disparity_color" stream. Then decoded data will include everything needed - w/h, type, ...

Comment (iamsg-luxonis) [reply include/depthai/host_data_packet.hpp:40]: I don't know why it was added here, but I agree, it should be added when it's created.
I will change it.

Comment (alex-luxonis) [reply include/depthai/host_data_packet.hpp]: Fixed.

**Pull request merged**